EasyPost/Http/Security.cs

```diff
@@ -0,0 +1,20 @@
+using System.Net;
+
+namespace EasyPost.Http
+{
+    internal static class Security
+    {
+        /// <summary>
+        ///     Get the TLS protocol version to use for the request.
+        /// </summary>
+        /// <returns>SecurityProtocolType TLS version enum.</returns>
+        internal static SecurityProtocolType GetProtocol()
+        {
+#if NET45
+            return SecurityProtocolType.Tls | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12;
+#else
+            return (SecurityProtocolType)0x00000C00;
```

Comment:
Issue: Nobody on planet Earth knows offhand what `0x00000C00` means... can we add a comment here with the human readable name of it?

Reply:
This is some legacy code, no longer relevant, not even used: https://stackoverflow.com/questions/47269609/system-net-securityprotocoltype-tls12-definition-not-found

Deleted.